### ISSUE TITLE
Assignment #2: issues with data 

### ISSUE THREAD

**Issue by kajajasik** [InseadDataAnalytics/INSEADAnalytics#122]
Hello, 

Has anyone found the solutions to the following problems with the data regarding the classification of payment status:

1/ categories -2 and 0 in PAY are not explained and the discussion on Kaggle does not give any answer. we tried to figure out the meaning from the billing and payment amounts, but we are not getting any coherent results. 
Do you guys have any ideas ?

2/ How can a person who has no billed amount from April to September in the whole period default on their payments in the next month (eg. ID 46) ?

**Comment by jiterra** [comment]
Hi,

I don't have answers to those, unfortunately. But wanted to add an extra question to the discussion.

What do the negative billing amounts mean? 

Cheers

**Comment by kajajasik** [comment]
In real banking situation it means that you were recredited (e.g. you returned something to the shop and got money back). Thus you credit card balance is negative and the bank owes you money. 
This amount should be then recredited to you (i.e. bank pays) or substracted from your next mont credit card payment.

**Comment by jiterra** [comment]
Yeah, that make sense. But if that is what it means then how come there are cases with negative billing (bank owes them) and default=1?

**Comment by bigSwingingDuck** [comment]
-2: No consumption; -1: Paid in full; 0: The use of revolving credit; 1 = payment delay for one month; 2 = payment delay for two months; . . .; 8 = payment delay for eight months; 9 = payment delay for nine months and above.

**Comment by kajajasik** [comment]
@jiterra I agree, it does not make sense, but there are more things that are weird in this data... e.g people jumping from being 8 months late to 2 moths late without any payment on their side registered in the meantime
@bigSwingingDuck thanks, I think it is the right intuition, even though it does not explain 100% of cases

**Comment by WiegerVos** [comment]
@kajajasik were you able to figure out how it works with the payment and bill amounts vs the payment timings?
